workshop.md

```diff
@@ -197,21 +212,22 @@ adjusts the targets. To configure a job with DNS service discovery, add the
 following to `prometheus.conf`:
 
 ```
-job {
-  name: "node"
-  sd_name: "demo-node.prometheus.io" # The SRV name.
-  metrics_path: "/metrics" # Need it separate as is not part of the SRV record.
-}
+- job_name: 'discovered_node'
+  dns_sd_configs:
+    - names:
+        - 'demo-node.prometheus.io'
 ```
 
 # The expression language
 
 With more metrics collected by your Prometheus server, it is time to
-familiarize yourself a bit more with the expression language. For comprehensive
-documentation, see http://prometheus.io/docs/querying/basics/. The following is
-meant as an inspiration for how to play with the metrics currently collected by
-your server. Evaluate them in the *Console* and *Graph* tab. For the latter, try
-different time ranges and the *stacked* option.
+familiarize yourself a bit more with the expression language. For
```

Comment:
We might need to standardize on a word-wrapping length or we will go mad... my vim by default uses 79 characters per line + newline.